frontend/src/reviews/ReviewsListContainer.tsx

```diff
@@ -0,0 +1,59 @@
+import { Container, Divider, List, Stack } from '@mui/material'
+import router from 'next/router'
+import { useEffect, useState } from 'react'
+import SimpleListItemButton from 'src/common/SimpleListItemButton'
+import ReviewsList from 'src/reviews/ReviewsList'
+import { ReviewKind, ReviewKindKeys, ReviewListStatusKeys } from 'types/types'
+
+function isReviewListContainerCategory(value: string | string[] | undefined): value is ReviewKindKeys {
```

Comment:
As this is a type guard for checking if it's a review kind, consider renaming to `isReviewKind`, moving into `types.ts` (next to the relevant type) and changing `value` to be `value: unknown` to make it more generic.